Userland/Libraries/LibJS/Runtime/VM.cpp

```diff
@@ -823,6 +823,29 @@ ThrowCompletionOr<void> VM::link_and_eval_module(Module& module)
     return {};
 }
 
+static String resolve_filepath(StringView filepath, StringView module_type)
+{
+    auto extensions = Vector<StringView> { "js"sv, "mjs"sv };
```

Comment:
```suggestion
    auto extensions = Vector<StringView,2> { "js"sv, "mjs"sv };
```
Lets not allocate here